### PR TITLE
Add similar projects, sponsors to readme

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -192,3 +192,19 @@ You can translate the game into other languages by editing the language files in
 - [Facebook](https://www.facebook.com/OpenRCT2)
 - [RCT subreddit](http://www.reddit.com/r/rct/)
 - [OpenRCT2 subreddit](http://www.reddit.com/r/openrct2/)
+
+## Similar Projects
+
+- [openage](https://github.com/SFTtech/openage)
+- [OpenLoco](https://github.com/OpenRCT2/OpenLoco)
+- [OpenRA](https://github.com/OpenRA/OpenRA)
+- [OpenTTD](https://www.openttd.org)
```
Comment (IntelOrca) [inline readme.md]: Is it possible to stick their logos in a table?

Comment (janisozaur) [reply readme.md]: | OpenRA                                                                                                        |   |   |   |   |
|---------------------------------------------------------------------------------------------------------------|---|---|---|---|
|  |   |   |   |   |
| Red Alert                                                                                                     |   |   |   |   |
| https://github.com/OpenRA/OpenRA                                                                              |   |   |   |   |

Like this?

**Comment by AaronVanGeffen** [comment]
Should we add some words of thanks for GitHub, Travis, and AppVeyor as well? They're a huge part of our infrastructure, even if they don't explicitly sponsor us.

File: readme.md (hunk 1 above)
Comment (Broxzier) [reply readme.md]: On desktop with full width they are not centered, which I think would look better.
image

**Comment by LRFLEW** [comment]
It's not like I want to overly self-promote, but [OpenMC2](https://github.com/LRFLEW/OpenMC2) is a related project. 😜

**Comment by janisozaur** [comment]
Seeing there are no further objections, I plan on merging that later today, after some history cleanup.

**Comment by Krutonium** [comment]
Should it say 

>7 Sponsors

?

I mean, it only lists four sponsors... I am going to guess this is a typo?

**Comment by janisozaur** [comment]
selection_002

**Comment by Krutonium** [comment]
Oh... >.<

**Comment by IntelOrca** [comment]
@Krutonium I made the same mistake.

**Pull request merged**